src/Components/ServiceScene/Breakdowns/FieldValuesBreakdownScene.tsx

```diff
@@ -0,0 +1,231 @@
+import {
+  PanelBuilders,
+  SceneComponentProps,
+  SceneCSSGridLayout,
+  SceneDataProvider,
+  SceneDataState,
+  SceneFlexItem,
+  SceneFlexLayout,
+  sceneGraph,
+  SceneObjectBase,
+  SceneObjectState,
+  SceneReactObject,
+} from '@grafana/scenes';
+import { buildDataQuery, LokiQuery } from '../../../services/query';
+import { getSortByPreference } from '../../../services/store';
+import { DataQueryError, LoadingState } from '@grafana/data';
+import { LayoutSwitcher } from './LayoutSwitcher';
+import { getQueryRunner } from '../../../services/panel';
+import { ByFrameRepeater } from './ByFrameRepeater';
+import { Alert, DrawStyle, LoadingPlaceholder } from '@grafana/ui';
+import { getFilterBreakdownValueScene } from '../../../services/fields';
+import { getLabelValue } from './SortByScene';
+import { getFieldGroupByVariable, VAR_FIELDS } from '../../../services/variables';
+import React from 'react';
+import {
+  FIELDS_BREAKDOWN_GRID_TEMPLATE_COLUMNS,
+  FieldsBreakdownScene,
+  getFieldBreakdownExpr,
+} from './FieldsBreakdownScene';
+import { AddFilterEvent } from './AddToFiltersButton';
+import { navigateToDrilldownPage } from '../../../services/navigate';
+import { PageSlugs } from '../../../services/routing';
+import { ServiceScene } from '../ServiceScene';
+import { DEFAULT_SORT_BY } from '../../../services/sorting';
+
+export interface FieldValuesBreakdownSceneState extends SceneObjectState {
+  body?: LayoutSwitcher | SceneReactObject;
+  $data?: SceneDataProvider;
+  lastFilterEvent?: AddFilterEvent;
+}
+export class FieldValuesBreakdownScene extends SceneObjectBase<FieldValuesBreakdownSceneState> {
+  constructor(state: Partial<FieldValuesBreakdownSceneState>) {
+    super(state);
+    this.addActivationHandler(this.onActivate.bind(this));
+  }
+
+  onActivate() {
+    const groupByVariable = getFieldGroupByVariable(this);
+    const tagKey = String(groupByVariable.state.value);
+    const query = buildDataQuery(getFieldBreakdownExpr(tagKey), { legendFormat: `{{${tagKey}}}`, refId: tagKey });
+
+    this.setState({
+      body: this.build(query),
+      $data: getQueryRunner([query]),
+    });
+
+    this._subs.add(
+      this.subscribeToEvent(AddFilterEvent, (event) => {
+        this.setState({
+          lastFilterEvent: event,
+        });
+      })
+    );
+
+    this._subs.add(
+      this.state.$data?.subscribeToState((newState) => {
+        this.onValuesDataQueryChange(newState, query);
+      })
+    );
+  }
+
+  private onValuesDataQueryChange(newState: SceneDataState, query: LokiQuery) {
+    if (newState.data?.state === LoadingState.Done) {
+      // No panels for the user to select, presumably because everything has been excluded
+      const event = this.state.lastFilterEvent;
```

Comment:
I like the concept, but not the fact that we're storing event references and using that to infer behaviors. Wouldn't it be almost the same if we just redirect when `newState.data.series.length < 1`?

Reply:
Like:

https://github.com/user-attachments/assets/a30a5406-3ca6-4c6f-b757-ec14b9d9398f

Reply:
Almost the same yes, but then if a user changes the variables up top, or happens to refresh the page and get no parsed results, they'll get an unexpected redirect. I wanted redirection to only happen on click of the last include or exclude button. Otherwise users should be presented with the "no fields, clear" UI instead of being moved around for reasons unknown to the user.